acceptance/bundle/variables/file-defaults/script

```diff
@@ -14,11 +14,11 @@ trace BUNDLE_VAR_cluster_key=mlops_stacks-cluster-overriden $CLI bundle validate
 title "variable has value in config file"
 trace $CLI bundle validate -o json --target with_value | jq $cluster_expr
 
-# title "file cannot be parsed"
-# trace errcode $CLI bundle validate -o json --target invalid_json | jq $cluster_expr
+title "file cannot be parsed"
+trace errcode $CLI bundle validate -o json --target invalid_json | jq $cluster_expr | sed 's/\\/\//g'
```

Comment:
This can be fixed with:
```
trace errcode $CLI bundle validate -o json --target invalid_json 2>&1 | sed 's/\\/\//g'
```

The jq expression is not needed because the command fails. The error is written to stderr so we need a redirect.